src/guides/hyperlinks.md

```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, [slash command](../guides/slashCommands.md) response content or ephemeral [`$onInteraction`](../callbacks/onInteraction.md) response content.
```

Comment:
Wrong line with the `or`. In this the `and` was perfect.

Reply:
Ouch, I overlooked.